docs/mkdocs.yml

```diff
@@ -42,7 +52,7 @@ extra_javascript:
 edit_uri: edit/main/docs/
 
 # Documentation and theme
-copyright: 'Copyright  &copy; 2024 Elyra Authors, an open source project and all are welcome to contribute! Maintained by the Elyra Authors Team'
+copyright: 'Copyright  &copy; 2024 Elyra Authors, an open source project! Maintained by the Elyra Authors Team'
```

Comment:
This is also a change I made. This shouldn't show as your change.

Reply:
This change is in 12.x branch as in this mkdocs.yml above change was not present so to keep it consistent with v13 I have updated this, hope this is fine.